serializations.md

```diff
@@ -245,6 +245,7 @@ subsequent sections divided by object.
 , <contract_address> :: binary()
```

Comment:
Is the nonce field expected here? it looks like there are two nonces here (while code has one https://github.com/aeternity/epoch/blob/098fcfad68adc13fac7842103de299c20ffbc22d/apps/aecontract/src/aect_call.erl#L133 ).